packages/add/README.md

```diff
@@ -0,0 +1,17 @@
+# webpack-cli - Add
+
+## Description
+
+This package contains the logic to add new properties to the a webpack configuration file.
+
+## Installation
+
+```bash
+npm i -D webpack-cli @webpack-cli/add
+```
+
+## Usage
+
+```bash
+webpack add
+```
```

Comment:
Maybe a screenshot would be good, as well as using it programatically. Could you do the same for the info pkg?

Reply:
Yes! I also wonder if there's need of more information inside the description?

Reply:
Some things like a screenshot, what we're asking for etc.. would be nice

Reply:
Any update on this?